app/command_autocomplete.go

```diff
@@ -216,6 +219,13 @@ func parseInputTextArgument(arg *model.AutocompleteArg, parsed, toBeParsed strin
 		}
 		return false, parsed + in[:indexOfSecondQuote+offset], in[indexOfSecondQuote+offset:], model.AutocompleteSuggestion{}
 	}
+	if isLast {
```

Comment:
Shouldn't this check be on top of the `if in[0] == '"' { `? 
Idea being if it's a last argument we do not care about quotes and with or without them we should return a valid suggestion. Thoughts?

Reply:
We still want the quoted behavior to be the same, and not short-circuit on a trailing space. Because the quotes are enclosing the spaces, we don't have to dodge the trailing space in the kludgey way we are on line 223.

Reply:
Makes sense!